src/main/resources/static/common/js/common.js

```diff
@@ -623,15 +623,17 @@ function connectAndSubscribeMQ(role, $http, $rootScope, $filter, mqConsumeCallba
                 var status = translatedStatus(msgObject.status, $filter);
 
                 if(role === ROLE_PURCHASER){    // purchaser produced this message to this topic
-                     toastr.success(requisition+' '+msgObject.orderNumber+' '+status+'!', '', {timeOut: 4000});
-                     if(mqProduceCallback){mqProduceCallback();}
+                     if (msgObject.requestedBy === CURRENT_USERNAME) {
+                         toastr.success(requisition+' '+msgObject.orderNumber+' '+status+'!', '', {timeOut: 4000});
+                         if(mqProduceCallback){mqProduceCallback();}
+                         // update the number on the icon
+                         getUnreviewedAmount($http,$rootScope,$filter);
+                         $rootScope.totalAmount = 0;
+                     }
                 }else if(role === ROLE_APPROVER){ // approver consume this message from this topic
                      toastr.info(requisition+' '+msgObject.orderNumber+' '+status+'!', '', {timeOut: 0});
```

Comment:
I believe that if I logged in with one approver and use SOAtest with another approver to approve the order, I'll see toastr for the second approver too. We need to add the logic to approver role too.